src/common.h

```diff
@@ -120,7 +120,8 @@ typedef enum
 {
     mem_storage_type_undefined,
     mem_storage_type_flash,
-    mem_storage_type_sd
+    mem_storage_type_sd,
+    mem_storage_type_ota_update

```

Comment:
fb_esp_mem_storage_type is the enum for internal and external storage for filesystems e.g. spiffs and littlefs for internal and sd and sd_mmc for external.

I think add mem_storage_type_ota_update here is irrelevant.